samples/react/ReactGrid/ReactApp/components/PeopleGrid.jsx

```diff
@@ -5,14 +5,16 @@ import { fakeData } from '../data/fakeData.js';
 import { columnMeta } from '../data/columnMeta.jsx';
 const resultsPerPage = 10;
 
+const fakeDataWithAction = fakeData.map(data => Object.assign(data, {actions: ''}));
```

Comment:
In latest version of Griddle, nothing is rendered for a cell if data does not have a property matching the column name, so had to add an empty field "actions".